community/values/src/main/java/org/neo4j/values/result/QueryResult.java

```diff
@@ -17,18 +17,38 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.bolt.v1.messaging.infrastructure;
+package org.neo4j.values.result;
```

Comment:
This move is a little bit mystic to me. I'm not saying it's bad, I would just like to understand the reason, as the interface it not clearly values related. Also some javadoc would be nice.